Workbooks/Usage/Active Users/es-es/Active Users.workbook

```diff
@@ -0,0 +1,185 @@
+      },
+      "conditionalVisibility": null
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "query": "",
+        "crossComponentResources": [],
+        "parameters": [
+          {
+            "id": "593d7ebe-107d-4823-87dd-d020d2138611",
+            "version": "KqlParameterItem/1.0",
+            "name": "Metric",
+            "type": 2,
+            "isRequired": true,
+            "query": "datatable(val: string, text: string)[\r\n'1d', 'Usuarios Activos por día',\r\n'7d', 'Usuarios Activos por semana',\r\n'28d', 'Usuarios Activos por mes (28d)',\r\n'30d', 'Usuarios Activos por mes (30d)',\r\n]",
+            "value": "28d",
+            "isHiddenWhenLocked": false,
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "timeContextFromParameter": null,
+            "resourceType": "microsoft.insights/components"
+          },
+          {
+            "id": "25a6fdc1-ae28-4886-9a5c-90254deb6a3d",
+            "version": "KqlParameterItem/1.0",
+            "name": "Activities",
+            "type": 2,
+            "description": "Un conjunto de vistas de página o eventos personalizados separado por comas. Este conjunto se va a usar para filtrar los eventos que cuentan como actividades. O usa '*' para incluir todos los eventos.",
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "union customEvents, pageViews\n| where timestamp >= ago(7d)\n| summarize count() by name\n| order by count_ desc\n| project Id=name, Title=name, Selected=false\n| union (\ndatatable(Id:string, Title:string, Selected:boolean)[\n'*', 'Todos los eventos y vistas de página', true,\n'%', 'Todas las vistas de página', false,\n'#', 'Todos los eventos personalizados', false\n]\n)",
+            "isHiddenWhenLocked": false,
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "timeContextFromParameter": null,
+            "resourceType": "microsoft.insights/components"
+          },
+          {
+            "id": "79126a9c-4ec6-4719-872a-f4a9b022e8f9",
+            "version": "KqlParameterItem/1.0",
+            "name": "OtherFilters",
+            "type": 1,
+            "description": "Un snippet de KQL que filtra aún más las actividades. Ejemplo: | where application_Version == '1.0'",
+            "isRequired": false,
+            "isHiddenWhenLocked": false,
+            "timeContextFromParameter": null
+          },
+          {
+            "id": "b911209e-14fc-4e33-b62a-f029f213ac2a",
+            "version": "KqlParameterItem/1.0",
+            "name": "TimeRange",
```

Comment:
Since you are in the template, can you cover this parameter to a true time range parameter instead of a drop down parameter?

Reply:
And make the corresponding change in the invariant template